src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  padding: 10px;
+
+  @each $day, $index in $days {
+    &--start-day--#{$day} &__day:first-child {
+      margin-left: $cell-shift * $index;
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    position: relative;
+    width: 100px;
+    height: 100px;
+    background-color: #eee;
+    border: 1px solid black;
```

don't forget to create variables for all colors

```suggestion
    border: 1px solid $c-black;
```